src/App.js

```diff
@@ -4,10 +4,14 @@ import AddTodoForm from './AddTodoForm';
 
 
 function App() {
+  const [newTodo, setNewTodo] = React.useState('');
   return (
     <div>
       <h1>Todo List</h1>
-      <AddTodoForm/>
+      <AddTodoForm onAddTodo={setNewTodo}/>
+        <p>
+          {newTodo}
+        </p>
```

Comment:
Minor spacing issue here. There's no reason for the `<p>` tags to be indented further than the other components, they should all be in line as `<p>` is not a child of `<AddTodoForm>`.

Also I would keep the `<p>` all on one line as there is only one thing inside and the line is still very short: `<p>{newTodo}</p>`